src/components/App.jsx

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
+
+import BookList from '../containers/BooksList';
+import BookForm from '../containers/BooksForm';
+
+export default class App extends Component {
+  constructor(props) {
+    super(props);
+    this.test = '';
```

Comment:
Why is the variable needed, more importantly, why is it a class variable? :thinking:

Reply:
Also, see other classes where this is and correct if there is no need for it, or state the use case as a reply